2022Swerve/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -44,8 +41,8 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
-    new JoystickButton(driverJoystick, OIConstants.kDriverZeroButtonID).whenPressed(() -> swerveSubsystem.zeroHeading());
-    new JoystickButton(driverJoystick, OIConstants.kDriverFODButtonID).whenPressed(() -> swerveSubsystem.toggleOrientation());
+    OI.Driver.getOrientationButton().whenPressed(() -> swerveSubsystem.toggleOrientation());
+    OI.Driver.getZeroButton().whenPressed(() -> swerveSubsystem.zeroHeading());
```

Comment:
I like this.  Looks clean

Reply:
This is an artifact from bypassing the OI PR